### ISSUE TITLE
[HL1S] [Steampipe] Map c2a5 Bug 139

### ISSUE THREAD

**Issue by Pinhead-alejandro** [Theuaredead/Half-Life-Source-Update#143]
Well, I see this like an intentional difference, (hi I have returned), but I will report it:

When you open the first door after start the chapter "Surface tension", you can notice that it can't be closed automatically, like happens in Half-Life. I would like explain that I see this like a intentional difference, since the button works perfectly in all ways. And we must understand that the malfunction from other doors are directly related with the buttons. Therefore, if this button works fine, maybe VALVe wanted to mantain these door opened. However, I am reporting it because in this case I could prefer the original configuration, like you will I think:

Pictures from Half-Life:

![2015-10-14_00001](https://cloud.githubusercontent.com/assets/13278870/10491558/80220516-72a7-11e5-8b88-f20974d22bc8.jpg)

![2015-10-14_00002](https://cloud.githubusercontent.com/assets/13278870/10491559/8025dfec-72a7-11e5-80d8-d925aafe8e77.jpg)

![2015-10-14_00003](https://cloud.githubusercontent.com/assets/13278870/10491560/8029eba0-72a7-11e5-920b-a638f2b387c4.jpg)

Picture from Half-Life: Source:

![2015-10-14_00001](https://cloud.githubusercontent.com/assets/13278870/10491599/b5a0acb0-72a7-11e5-803b-1404870f4267.jpg)

**Comment by Theuaredead** [comment]
Yeah, that's a bug, it isn't going to the Green light correctly, I can't tell if this is a VMT issue or a mapping issue.

**Comment by Pinhead-alejandro** [comment]
Wait, the button works exactly like it works in Half-Life. It pass from red to green and from green to red. That is correct. But the door, can't be closed alone in Half-Life: Source, like it should, like it happens in Half-Life.
